### ISSUE TITLE
No diacritics or special characters support?

### ISSUE THREAD

**Issue by work79** [trelby/trelby#8]
Writing special characters such as: `ěščřž` does not work.

Is there a way to change encoding or something like that?

**Comment by work79** [comment]
For anyone stumbling upon this issue, Trelby supports only ISO-8859-1 (Latin-1) due to some PDF export problems.
More info:
https://web.archive.org/web/20201226153814/https://github.com/oskusalerma/trelby/issues/85

I think there will be no UTF-8 support due to old codebase being unmaintainable (python2, old PDF libraries).

**Comment by JusSolo** [comment]
¿Wats about spanish tiles: á é í ó ú and ñ?

**Comment by janopae** [comment]
We fixed this issue actually in a fork of Trelby, that continued development after 2016. It's https://github.com/limburgher/trelby.

Currently, we only provide installables for Linux, but I recently added a Windows dual boot to my machine and I want to look into creating a Windows installer in the future. (If any one else here has experience with creating Windows installers for Python apps, your help would be appreciated!)

I also want to look into making it work on MacOS sooner or later.
